### PR TITLE
Vine Plotting Tool vine_plot_txn_log

### PR DESCRIPTION
**Pull request by BarrySlyDelgado**

### CODE REVIEW

File: taskvine/src/tools/vine_plot_txn_log.py [hunk 1]
```diff
@@ -0,0 +1,331 @@
+from collections import Counter
```
Comment (btovar) [inline taskvine/src/tools/vine_plot_txn_log.py:1]: First line should be: 
```sh
#! /usr/bin/env python
```

**Review by btovar — CHANGES_REQUESTED** [review]

File: taskvine/src/tools/vine_plot_txn_log.py [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+import matplotlib.pyplot as plt
```
Comment (btovar) [inline taskvine/src/tools/vine_plot_txn_log.py:2]: Also, add a copyright notice.

**Review by btovar — CHANGES_REQUESTED** [review]

**Comment by dthain** [comment]
`vine_graph_txn_log` for consistency with other tools.

**Review by btovar — APPROVED** [review]

File: taskvine/src/tools/vine_plot_txn_log [hunk 3]
```diff
@@ -0,0 +1,467 @@
...
+# This software is distributed under the GNU General Public License.
+# See the file COPYING for details.
+
+from collections import Counter
+import matplotlib.pyplot as plt
+import time
+import sys
+import argparse
+import json
+
```
Comment (btovar) [inline taskvine/src/tools/vine_plot_txn_log:13]: Let's add this style for accessibility:

`plt.style.use('tableau-colorblind10')`

**Review by btovar — CHANGES_REQUESTED** [review]

**Review by btovar — APPROVED** [review]

**Comment by btovar** [comment]
Ready to be merged?

**Comment by BarrySlyDelgado** [comment]
Its ready.

**Pull request merged**